### PR TITLE
LibSVM build using Wizard

### PR DESCRIPTION
**Pull request by aviks**

### CODE REVIEW

File: L/libsvm/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+make 
+make lib
+if [[ "${target}" == *mingw32 ]]; then
+	cp libsvm.${dlext} ${prefix}/bin
+else
+	cp libsvm.${dlext} ${prefix}/lib
+fi
+cp svm-train${exeext} ${prefix}/bin
+cp svm-predict${exeext} ${prefix}/bin
+cp svm-scale${exeext} ${prefix}/bin
```
Comment (giordano) [inline L/libsvm/build_tarballs.jl:36]:
```suggestion
cp libsvm.${dlext} ${libdir}
cp svm-train${exeext} ${bindir}
cp svm-predict${exeext} ${bindir}
cp svm-scale${exeext} ${bindir}
```

File: L/libsvm/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+products = [
+    LibraryProduct("libsvm", :libsvm),
+    ExecutableProduct("svm-scale", :svm_scale),
+    ExecutableProduct("svm-train", :svm_train),
+    ExecutableProduct("svm-predict", :svm_predict)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
```
Comment (giordano) [inline L/libsvm/build_tarballs.jl:49]: I just noticed this: why was this needed? :thinking:

Comment (aviks) [reply L/libsvm/build_tarballs.jl:49]: It tries to build with openmp. I'm not sure it works, but it's theoretically needed.

Comment (giordano) [reply L/libsvm/build_tarballs.jl:49]: I see, thanks!
